core/src/main/scala/fr/davit/akka/http/metrics/core/HttpMetricsRegistry.scala

```diff
@@ -58,6 +58,7 @@ object HttpMetricsRegistry {
     }
   }
 
+  final case class CustomDimension(key: String, value: String) extends Dimension {}
```

Comment:
I think this is better not to provide any implementation. User should extend `Dimension` themselves

Reply:
I will add this to the docs.